src/common/EventHandlers.js

```diff
@@ -48,8 +48,14 @@ import { INVERSE_MOTION_THRESHOLD,INVERSE_ROTATION_THRESHOLD, MOTION_THRESHOLD,
                 body.rotate(0, INVERSE_ROTATION_THRESHOLD, 0);
                 break;
             case 'q':
-                body.setPosition(0, 0, 0);
-                body.setRotation(0, 0, 0);
+                body.setPosition(-3, 4, 0);  // Corrected position
```

Comment:
Will fix this when making the 3D model on blender. This will be important for the ML side as well, to standardize the coordinates